src/main/webapp/WEB-INF/pages/admin/user.ftl

```diff
@@ -48,17 +48,6 @@
                     <div class="col-md-6">
                         <@select name="user.role" value=user.role javaGetter=false options={"User":"user.roles.user", "Manager":"user.roles.manager", "Publisher":"user.roles.publisher", "Admin":"user.roles.admin"}/>
                     </div>
-
-                    <#if "${newUser!}"=="no">
-                        <div class="userPasswordButtons">
-                            <@label i18nkey="user.password">
-                                <@s.submit cssClass="button" name="resetPassword" key="button.resetPassword" />
-                            </@label>
-                        </div>	
-                    <#else>
```

Comment:
Check if these lines are needed to assign password to new user because with this PR the administration function does not work when you create a new user.

Reply:
Done. I revert this change partially to remove duplicated reset password